### PR TITLE
build: Update shader toolchain

### PR DESCRIPTION
**Pull request by mikes-lunarg**

### CODE REVIEW

File: layers/vulkan/generated/feature_requirements_helper.cpp [hunk 1]
```diff
@@ -4357,21 +4357,21 @@ FeatureAndName AddFeature(APIVersion api_version, vkt::Feature feature, void **i
...
-            auto vk_struct = const_cast<VkPhysicalDevicePortabilitySubsetFeaturesKHR *>(
-                vku::FindStructInPNextChain<VkPhysicalDevicePortabilitySubsetFeaturesKHR>(*inout_pnext_chain));
-            if (!vk_struct) {
-                vk_struct = new VkPhysicalDevicePortabilitySubsetFeaturesKHR;
-                *vk_struct = vku::InitStructHelper();
-                if (*inout_pnext_chain) {
-                    vvl::PnextChainAdd(*inout_pnext_chain, vk_struct);
-                } else {
-                    *inout_pnext_chain = vk_struct;
+                case Feature::constantAlphaColorBlendFactors : {
```
Comment (spencer-lunarg) [inline layers/vulkan/generated/feature_requirements_helper.cpp:4360]: ... my clang-format reverted this... will need to figure out afterwards what is going on with clang-format here

**Review by spencer-lunarg — APPROVED** [review]

File: layers/vulkan/generated/feature_requirements_helper.cpp (hunk 1 above)
Comment (mikes-lunarg) [reply layers/vulkan/generated/feature_requirements_helper.cpp:4360]: Hmm probably a version difference. I did the update with `Ubuntu clang-format version 14.0.0-1ubuntu1.1`

**Comment by arno-lunarg** [comment]
glslang now requires CMake 3.27, causing build failures : /

**Comment by spencer-lunarg** [comment]
closing for https://github.com/KhronosGroup/Vulkan-ValidationLayers/pull/9039

**Pull request closed without merging**